### PR TITLE
EquipSwitch Packet

### PR DESCRIPTION
**Pull request by sctnightcore**
**Packet base IRO:RE**
rA PR : https://github.com/rathena/rathena/pull/3548
ty to rA dev ♥
**Allow edits from maintainers. If you are interested in developing **
```
[Send]
'0A97' => 'equipswitch_add'
'0A99' => 'equipswitch_remove'
'0A9C' => 'equipswitch_run'
'0ACE' => 'equipswitch_single'
```
```
[Recv]
'0A98' => 'equip_item_switch',
'0A9A' => 'unequip_item_switch'
'0A9D' => 'equipswitch_run_res'
'0A9B' => 'equipswitch_log'
```
image
image

### CODE REVIEW

**Comment by Nipodemos** [comment]
ready to merge?

**Comment by sctnightcore** [comment]
@Nipodemos  Wait ... someone review it .

**Comment by Fadreus** [comment]
Adding console command to use Swap Gear? :no_mouth:

**Comment by sctnightcore** [comment]
@Fadreus  I'm lazy

**Comment by Fadreus** [comment]
Mm.. K.. :no_mouth: 
image

**Comment by Fadreus** [comment]
#2676

File: src/Network/Receive.pm [hunk 1]
```diff
@@ -8017,24 +8017,40 @@ sub cash_buy_fail {
 sub equip_item {
 	my ($self, $args) = @_;
 	my $item = $char->inventory->getByID($args->{ID});
-	if ((!$args->{success} && $args->{switch} eq "00AA") || ($args->{success} && $args->{switch} eq "0999")) {
-		message TF("You can't put on %s (%d)\n", $item->{name}, $item->{binID});
+	if ((!$args->{success} && $args->{switch} eq "00AA") || ($args->{success} && $args->{switch} eq "0999") || (($args->{success} == 1 || $args->{success} == 2 ) && $args->{switch} eq "0A98")) {
+		if ($args->{switch} == '0A98') {
```
Comment (sctnightcore) [inline src/Network/Receive.pm]: ($args->{switch} == '0A98' ) ? $item->{eqswitch} = $args->{type} : $item->{equipped} = $args->{type};

 
Maybe?

**Pull request merged**